src/main/fc/fc_msp.c

```diff
@@ -142,13 +142,13 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT] = {
     { BOXSONAR, "SONAR", 22 },
     { BOXSERVO1, "SERVO1", 23 },
     { BOXSERVO2, "SERVO2", 24 },
-    { BOXSERVO3, "SERVO3", 25 },
     { BOXBLACKBOX, "BLACKBOX", 26 },
     { BOXFAILSAFE, "FAILSAFE", 27 },
     { BOXAIRMODE, "AIR MODE", 28 },
-    { BOX3DDISABLESWITCH, "DISABLE 3D SWITCH", 29},
-    { BOXFPVANGLEMIX, "FPV ANGLE MIX", 30},
+    { BOX3DPOSSWITCH, "POSTIVE ONLY 3D", 29},
+    { BOX3DNEGSWITCH, "NEGITIVE ONLY 3D", 30},
```

Comment:
Actually just one extra mode.  I had renamed BOX3DDISABLESWITCH to BOX3DPOSWITCH (only pos throttle values) and added the BOX3DNEGSWITCH (only neg throttle values) to go from 1500 to 1000.

I was trying to do the 3dmode to BOX3DDISABLESWITCH, but I was having issues accidentally going to 1000.  I wanted to limit the min throttle 1400 instead of 1000 but dshot appears to ignore that.   This way is safer in that low throttle is going to be low regardless of positive or negative.  It also could be a way to fly 3d by using a toggle instead of center throttle, I was messing with that in the yard just a bit and I liked it.

Oops spelling - I had quick hacked it up to test this...